### PR TITLE
Reactnative/upgrade react native

### PR DESCRIPTION
**Pull request by IbrahimSulai**
Upgraded the react-native version to 0.62.0 and also integrated the react-native flipper for debug mode.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/3911)

### CODE REVIEW

File: source/community/reactnative/.eslintrc.js  [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
```
Comment (pragadeeshk) [inline source/community/reactnative/.eslintrc.js :1]: Is this file required to be in GitHub?

File: source/community/reactnative/.flowconfig [hunk 2]
```diff
@@ -0,0 +1,74 @@
+[ignore]
```
Comment (pragadeeshk) [inline source/community/reactnative/.flowconfig:1]: Is this file required to be in GitHub?

File: source/community/reactnative/.gitattributes [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# specific for windows script files
```
Comment (pragadeeshk) [inline source/community/reactnative/.gitattributes:1]: Is this file required to be in GitHub?

File: source/community/reactnative/.prettierrc.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
```
Comment (pragadeeshk) [inline source/community/reactnative/.prettierrc.js:1]: Is this file required to be in GitHub?

File: source/community/reactnative/android/app/build.gradle [hunk 5]
```diff
@@ -121,17 +159,31 @@ android {
         }
     }
     buildTypes {
+        debug {
+            signingConfig signingConfigs.debug
+        }
         release {
+            // Caution! In production, you need to generate your own keystore file.
+            // see https://facebook.github.io/react-native/docs/signed-apk-android.
+            signingConfig signingConfigs.debug
```
Comment (pragadeeshk) [inline source/community/reactnative/android/app/build.gradle:168]: This might get missed. Better to provide a proper signingConfig or omit it instead of providing debug config

File: source/community/reactnative/ios/RNAdaptiveCards.xcodeproj/xcshareddata/xcschemes/RNAdaptiveCards-tvOS.xcscheme [hunk 6]
```diff
@@ -1,25 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "0940"
+   LastUpgradeVersion = "1130"
    version = "1.3">
    <BuildAction
-      parallelizeBuildables = "NO"
+      parallelizeBuildables = "YES"
```
Comment (pragadeeshk) [inline source/community/reactnative/ios/RNAdaptiveCards.xcodeproj/xcshareddata/xcschemes/RNAdaptiveCards-tvOS.xcscheme:6]: shouldn't this be "No"

File: source/community/reactnative/.eslintrc.js  (hunk 1 above)
Comment (IbrahimSulai) [reply source/community/reactnative/.eslintrc.js :1]: Have created a sample react native project with the latest react-native 0.62.0.

It includes .eslintrc.js file. I can also see the same configuration file in the other latest projects too.

File: source/community/reactnative/.flowconfig (hunk 2 above)
Comment (IbrahimSulai) [reply source/community/reactnative/.flowconfig:1]: Have created a sample react native project with the latest react-native 0.62.0.

It includes .flowconfig file. I can also see the same configuration file in the other latest projects too.

File: source/community/reactnative/.prettierrc.js (hunk 4 above)
Comment (IbrahimSulai) [reply source/community/reactnative/.prettierrc.js:1]: Have created a sample react native project with the latest react-native 0.62.0.

It includes .prettierrc.js file. I can also see the same configuration file in the other latest projects too.

File: source/community/reactnative/.gitattributes (hunk 3 above)
Comment (IbrahimSulai) [reply source/community/reactnative/.gitattributes:1]: Have created a sample react native project with the latest react-native 0.62.0.

It includes a .gitattributes file. They mentioned, Its need for the windows configuration.

File: source/community/reactnative/ios/RNAdaptiveCards.xcodeproj/xcshareddata/xcschemes/RNAdaptiveCards-tvOS.xcscheme (hunk 6 above)
Comment (IbrahimSulai) [reply source/community/reactnative/ios/RNAdaptiveCards.xcodeproj/xcshareddata/xcschemes/RNAdaptiveCards-tvOS.xcscheme:6]: For the tv-OS scheme - "parallelizeBuildables" should be YES.

File: source/community/reactnative/android/app/build.gradle (hunk 5 above)
Comment (IbrahimSulai) [reply source/community/reactnative/android/app/build.gradle]: Commented out this line.
